app-modules/inventory-management/src/Models/MaintenanceProvider.php

```diff
@@ -52,6 +52,7 @@ class MaintenanceProvider extends BaseModel implements Auditable
 
     protected $fillable = [
         'name',
+        'assets_count',
```

Comment:
This can be removed. You can get the details through the `maintenanceActivities` relationship.

It's also supposed to be activities, not assets.